client-scripts-api.md

```diff
@@ -40,11 +40,12 @@ The argument consists of the following fields
 - `channel` is used to send and receive messages using an interface similar to a [broadcast channel](https://developer.mozilla.org/en-US/docs/Web/API/Broadcast_Channel_API):
   ```javascript
   window.setTaskHook("recording", ({ channel }) => {
-    channel.onmessage = (event) => {
-      if (event.type === 'video-pause') {
+    // Add event listener
+    channel.on((event) => {
```

Comment:
💭  shall we add a filter for tracked in the example? as i feel that will be used by almost all our clients